03_letters/shahriys/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+var outletter io.Writer = os.Stdout
+
+func letters(s string) map[rune]int {
+
+	alphamap := make(map[rune]int)
+
+	for i := 0; i < len(s); i++ {
+		if val, ok := alphamap[rune(s[i])]; ok {
```

Comment:
`alphamap[key] += 1` will work even if `key` does not yet exist in that map ([example](https://goplay.space/#bSn7U4A9ZMc)). You can use this to get rid of the `if` block entirely.
👍 on using assignment in the `if` block though

Reply:
incorporated